### PR TITLE
Various QOL and functional changes to match new old intake

### PR DESCRIPTION
**Pull request by jbko6**
Changed some various variables to match drive team requests, added QOL per drive team request

### CODE REVIEW

**Comment by jbko6** [comment]
think i ported over most of the important stuff, may have missed a few things though

**Review by TAKBS2412 — COMMENTED** [review]
Does this include the arm subsystem cleanup? (Removing old arm feedforward, logging changes per #62, etc.). It would be good to have those.

**Comment by jbko6** [comment]
Now it does include those changes

File: src/main/java/frc/team2412/robot/subsystems/ArmSubsystem.java [hunk 1]
```diff
@@ -183,19 +162,19 @@ public static enum PositionType {
...
+	DoublePublisher shoulderPositionPublisher;
+	DoublePublisher elbowPositionPublisher;
+	DoublePublisher wristPositionPublisher;
 
 	DoublePublisher armPIDPublisher;
 	DoublePublisher armFeedforwardPublisher;
 
-	StringPublisher armGoalPublisher;
+	DoublePublisher armGoalPublisher;
 	StringPublisher armPositionPublisher;
```
Comment (jamesdooley4) [inline src/main/java/frc/team2412/robot/subsystems/ArmSubsystem.java:174]: Need to update this too

**Review by jamesdooley4 — DISMISSED** [review]
just one change needed, again

**Review by jamesdooley4 — APPROVED** [review]

**Pull request merged**